src/main/resources/avro/referencemethods.avdl

```diff
@@ -325,4 +396,99 @@ GetSequenceBasesResponse getSequenceBases(
     /** Additional request parameters to restrict the query. */
     GetSequenceBasesRequest request) throws GAException;
 
+/****************  /joins/search  *******************/
+/**
+This request maps to the body of `POST /joins/search`
+as JSON.
+*/
+record SearchJoinsRequest {
```

Comment:
This method seems pretty heavily overloaded and complicated. I think it would be better to have `SearchReferenceJoins` and `SearchVariantSetJoins`, so the semantics aren't so complicated.